frontend/src/styles/Team.css

```diff
@@ -33,6 +33,32 @@
     font-family: Arial, Helvetica, sans-serif;
     text-align: left;
   }
+  
+  .btn-group{
+    padding-top: 1vh;
+    width: 100%;
+    justify-content: center;
+  }
+  .btn-group button {
+    background-color: #333; 
```

Comment:
I recommend making the background-color into a variable in App.css to add to our color scheme. Everyone can then call the same color variable when they make a button next time from App.css. This will help us keep our color scheme cohesive.

Reply:
Done!